src/components/table/Table.js

```diff
@@ -951,13 +953,17 @@ class Table extends Component {
     }
   };
 
-  handleFieldEdit = (selected, fieldName) => {
-    this.closeContextMenu();
+  handleFieldEdit = () => {
+    const { selected, contextMenu } = this.state;
 
-    const selectedId = selected[0];
+    if (contextMenu.supportFieldEdit && selected.length === 1) {
```

Comment:
I believe selected can be undefined as well. Case in which this will trigger an error. Can you pls add a safety check for that as well or define it in the state as [] in constructor ? Thanks

Reply:
selected[0] can be undefined. Otherwise as far as I'm aware it should always be an array. It is defined as one in the state in `tableHelpers`.